vaccine_feed_ingest/stages/ingest.py

```diff
@@ -152,7 +157,12 @@ def run_parse(
         except CalledProcessError as e:
             if fail_on_runner_error:
                 raise e
-            logger.error("Subprocess errored, stage will be skipped: %s", e)
+            logger.error(
+                "Subprocess %s/%s errored, stage will be skipped: %s",
+                site_dir.parent.name,
+                site_dir.name,
+                e,
```

Comment:
Not quite sure how sentry works, but the python standard is to add `exc_info=True` if you want the exception in the log statement